integrations/gitlab.rst

```diff
@@ -0,0 +1,63 @@
+
+|gitlab_logo| Gitlab 
+=============================
+
+You can use `Gitlab CI`_ cloud or local service to automatically build and test your project in Linux/OSX/Windows environments.
+It is free for OSS projects, and offers an easy integration with Gitlab, so builds can be automatically
+fired in Gitlab CI after a ``git push`` to Gitlab.
+
+You can use Gitlab CI both for:
+
+- Building and testing your project, which manages dependencies with Conan, and probably a conanfile.txt file
+- Building and testing conan binary packages for a given conan package recipe (with a conanfile.py)
+
+
+Building and testing your project
+------------------------------------
+
+We are going to use the :ref:`Google Test example<google_test_example>` now, with **Gitlab CI** support to run the tests.
+
+
+Clone the project from github:
+
+
+.. code-block:: bash
+
+   $ git clone https://github.com/lasote/conan-gtest-example
+
+
+Create a ``.gitlab-ci.yml`` file and paste this code in it: 
+
+
+.. code-block:: text
+ 
+    image: lasote/conangcc63
+
+    build:
+      # Upgrade Conan version
+      before_script:
+        - sudo pip install --upgrade conan
+
+      script:
+        # Automatic detection of your arch, compiler, etc. Download dependencies
+        - conan install
```

Comment:
I think I much prefer the ``conan test_package`` approach, which is better for settings-build consistency. 
In fact, one of the big changes we did to the docs is to describe ``test_package`` as the main package creation method.

What do you think?